### ISSUE TITLE
Batch DVD Auto-Processing

### ISSUE THREAD

**Issue by GT40Wins** [RandomEngy/VidCoder#36]
As it stands, except for my earlier issue, the program works great.

Just wondering if there was a way to automatically process all of the DVD's in a folder without having to feed them to VidCoder one at a time?

Trying to save space by converting DVD files (ifo & vob ar 4+GB) to MP4 (<1GB) using VidCoder.  MP4's are also acceptable to Plex Server without trans-coding. 

Thank You

**Comment by RandomEngy** [comment]
Try dragging the parent folder onto the main window.

**Comment by GT40Wins** [comment]
THANKS !!!  I started dragging groups of folders and as long as there were not too many it would Auto-Select Title and then add to Que.

Not sure if there was a problem but when I dragged more that let's say 25 it seems to start processing then the progress bar freezes.  However was able to save Que and re-start VidCoder and start again.

During encoding of this same group it appeared to be stuck on the first one after 5 hrs, so restarted and at end of que found that it had been working reliably and during 5 hrs 39 had been encoded.

YAHOO !!!!
